ultralist/screen_printer.go

```diff
@@ -53,19 +53,29 @@ func (f *ScreenPrinter) Print(groupedTodos *GroupedTodos, printNotes bool) {
 	for _, key := range keys {
 		tabby.AddLine(cyan.Sprint(key))
 		for _, todo := range groupedTodos.Groups[key] {
-			f.printTodo(tabby, todo, printNotes)
+			f.printTodo(tabby, todo, printNotes, showStatus)
 		}
 		tabby.AddLine()
 	}
 	tabby.Print()
 }
 
-func (f *ScreenPrinter) printTodo(tabby *tabby.Tabby, todo *Todo, printNotes bool) {
-	tabby.AddLine(
-		f.formatID(todo.ID, todo.IsPriority),
-		f.formatCompleted(todo.Completed),
-		f.formatDue(todo.Due, todo.IsPriority, todo.Completed),
-		f.formatSubject(todo.Subject, todo.IsPriority))
+func (f *ScreenPrinter) printTodo(tabby *tabby.Tabby, todo *Todo, printNotes bool, showStatus bool) {
+	if showStatus {
+		tabby.AddLine(
+			f.formatID(todo.ID, todo.IsPriority),
+			f.formatCompleted(todo.Completed),
+			f.formatInformation(todo),
+			f.formatDue(todo.Due, todo.IsPriority, todo.Completed),
+			f.formatSubject(todo.Subject, todo.IsPriority))
+	} else {
+		tabby.AddLine(
+			f.formatID(todo.ID, todo.IsPriority),
+			f.formatCompleted(todo.Completed),
+			f.formatDue(todo.Due, todo.IsPriority, todo.Completed),
+			f.formatSubject(todo.Subject, todo.IsPriority))
+	}
```

Comment:
@timorunge I feel like I'm making a mess of this.  at some point I'll probably have to come back and refactor how todos are printed to the screen.